psi4/driver/procrouting/dft/libxc_functionals.py

```diff
@@ -37,6 +37,8 @@
 funcs.append({"name": "TETER93"        , "xc_functionals": {"LDA_XC_TETER93"            : {}}})
 funcs.append({"name": "ZLP"            , "xc_functionals": {"LDA_XC_ZLP"                : {}}})
 funcs.append({"name": "KSDT"           , "xc_functionals": {"LDA_XC_KSDT"               : {}}})
+funcs.append({"name": "LDA0"           , "xc_functionals": {"HYB_LDA_XC_LDA0"           : {}}})
+funcs.append({"name": "CAM-LDA0"       , "xc_functionals": {"HYB_LDA_XC_CAM_LDA0"       : {}}})
```

Comment:
The test is failing because you're trying to build these two functionals, which are not in the version of LibXC which is on travis. I'd uncomment the tests (if the functionals are not there, the tests are skipped) and comment these two lines. Hopefully we'll poll for the predefined "XC" functionals from LibXC at some point in the future...

Reply:
I would have thought the opposite. Thanks for the tip.